health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/Filters.js

```
@@ -0,0 +1,273 @@
+
+  const handleFilterChange = (data) => {
+    setValue({ ...value, ...data });
+  };
+
+  const handleCycleFilter = (data) => {
+    setValue({ ...value, filters: { ...value?.filters, cycle: data.cycle } });
+  };
+
+  const clearCycleFilter = () => {
+    const prevFilters = value?.filters;
+    delete prevFilters.cycle;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)